nncf/common/graph/utils.py

```diff
@@ -89,3 +91,57 @@ def get_split_axis(input_shapes: List[List[int]], output_shapes: List[List[int]]
         logger.warning('Identity split/concat node detected')
 
     return axis
+
+
+def check_scope_names_match_graph(config: NNCFConfig, graph: NNCFGraph) -> None:
```

Comment:
Are you going add support for: https://github.com/openvinotoolkit/nncf/blob/b39eca98fab82482e30846bb15ba826b6753d058/nncf/parameters.py#L48

Reply:
Not planned. IgnoredScopes supported in NNCFConfig?